src/resources/elements/ckEditor/pckeditor-text.ts

```diff
@@ -45,11 +63,18 @@ export class PCkeditorText {
           editor.editing.view.document.on("clipboardInput", (evt, data) => {
             const dataTransfer = data.dataTransfer;
             const textContent = dataTransfer.getData("text/plain");
-
+            const totalDataLength = textContent.length + editor.getData().length;
```

Comment:
Could you add few comments on how `dataTransfer` and `textContent` are different? (in the code)